src/win32/window_manager.cc

```diff
@@ -0,0 +1,44 @@
+#include "../window_manager.h"
+#include <windows.h>
+
+BOOL CALLBACK EnumWindowsProc(HWND hwnd, LPARAM lparam) {
```

Comment:
The interface of `EnumWindowsProc` is outdated since 2018 concerning [the documentation](https://docs.microsoft.com/en-us/previous-versions/windows/desktop/legacy/ms633498(v=vs.85)). On the other hand, I cannot find anything that is more up to date. So I think we could leave it like that but have to keep in mind that this is outdated. Maybe a comment to the docs would make sense in case future Simon or future Sven have to take action on an issue. :wink:

Reply:
I'm not sure about that. It mentions that the docs page is no longer updated regularly, but I can't find a deprecation notice.

Anyways, mentioning this in a little doc comment won't hurt us 👍

Reply:
I agree, there is no official deprecation warning. I just thought this might be the case due to the breadcrumbs. Maybe a little bit too much interpretation of the docs. :sweat_smile: 
![image](https://user-images.githubusercontent.com/25754312/89786424-30a66580-db1c-11ea-8b0d-85d91e52513c.png)